README.md

```diff
@@ -255,6 +255,21 @@ Adding your own asset transformation functions is fair game too. In
 fact, it's encouraged. Let's say you need to serve all assets from a
 Content Delivery Network ...
 
+#### Serving bundles under a custom URL prefix
```

Comment:
This section should probably not be between that ellipsis and it's continuation in the section below. :)